pkgs/development/python-modules/mailsuite/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, isPy3k, pythonPackages }:
+
+buildPythonPackage rec {
+  pname = "mailsuite";
+  version = "1.6.1";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "17bsnfjjzv8hx5h397p5pa92l6cqc53i0zjjz2p7bjj3xqzhs45a";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    dnspython
+    html2text
+    mail-parser
+    IMAPClient
+  ];
+
```

Comment:
checkPhase nad importsCheck is missing.

Reply:
Yeah, the tests are excluded from the PyPi package. The NixOS test should be more comprehensive anyway and if we want, we can add more test reports to it.

Reply:
```
  pythonImportsCheck = [ "mailsuite" ];
```

We still want a pythonImportsCheck and basic tests if there are some because when we are rebuilding depending packages the tests are not exectued.

Reply:
Oh, you're talking about mailsuite. It doesn't have any unit tests at all, so an import check would be good. I won't be able to fix this for a few days.